observability/kiali/ossm-console-plugin-assembly.adoc

```diff
@@ -0,0 +1,54 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-console-plugin-assembly"]
+= Using OpenShift Service Mesh Console plugin
+include::_attributes/common-attributes.adoc[]
+:context: ossm-console-plugin-assembly
+
+toc::[]
+
+The {SMPlugin} extends the {ocp-short-name} web console with a {SMProductShortName} menu and enhanced tabs for workloads and services.
+
+[IMPORTANT]
+====
+If your browser does not trust the certificate, you can manually accept it in the Kiali standalone UI before accessing the {SMPluginShort}.
+====
+
+include::modules/ossm-about-console-plugin.adoc[leveloffset=+1]
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-install-console-plugin_{context}"]
+== Installing {SMPlugin-s}
+
+You can install the {SMPluginShort} by using the {ocp-short-name} web console or the {ocp-short-name} CLI.
+
+[NOTE]
+====
+{SMPluginShort} is only supported on {ocp-short-name} 4.15 and above. For {ocp-short-name} 4.14 users, only the standalone Kiali console is accessible.
+====
+
+include::modules/ossm-install-console-plugin-ocp-web-console.adoc[leveloffset=+2]
+
+include::modules/ossm-install-console-plugin-ocp-cli.adoc[leveloffset=+2]
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-uninstall-console-plugin_{context}"]
+== Uninstalling {SMPlugin-s}
+
+You can uninstall the {SMPluginShort} by using the {ocp-short-name} web console or the {ocp-short-name} CLI.
+
+[NOTE]
+====
+You must uninstall the OSSMC plugin before removing the Kiali Operator. Deleting the Operator first may leave OSSMC and Kiali CRs stuck, requiring manual removal of the finalizer. Use the following command with `<CR type>` as `kiali` or `ossmconsole` to remove the finalizer, if needed:
+[source,terminal]
+----
+$ oc patch <CR type> <CR name> -n <CR namespace> -p '{"metadata":{"finalizers": []}}' --type=merge
+----
```

Comment:
user-replaced values are `<lowercase_with_underscores>`. Later, you're using e.g. `custom_resource_name`, should probably stay consistent:
```suggestion
You must uninstall the OSSMC plugin before removing the Kiali Operator. Deleting the Operator first may leave OSSMC and Kiali CRs stuck, requiring manual removal of the finalizer. Use the following command with `<custom_resource_type>` as `kiali` or `ossmconsole` to remove the finalizer, if needed:
[source,terminal]
----
$ oc patch <custom_resource_type> <custom_resource_name> -n <custom_resource_namespace> -p '{"metadata":{"finalizers": []}}' --type=merge
----
```